.github/workflows/test.yml

```diff
@@ -33,10 +33,10 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest, macOS-latest, windows-latest]
-        go_version: [1.17.x, 1.18.x, 1.19.x]
+        go_version: [1.17.x, 1.18.x, 1.19.x, 1.20.x]
     runs-on: ${{ matrix.os }}
     steps:
-      - name: Install Go ${{ matrix.go }}
+      - name: Install Go ${{ matrix.go_version }}
```

Comment:
nice catch!

Reply:
Thanks to https://marketplace.visualstudio.com/items?itemName=GitHub.vscode-github-actions which also checks for errors.